blog/2023-02-08-preflight-vs-inflight.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "The Great Divide: Preflight vs Inflight Resource Creation ⚔️"
+description: Why does Wing only let you create resources in preflight?
+authors: 
+  - rybickic
+tags: [cloud-oriented programming, winglang, resources, preflight, inflight, iac, security]
+hide_table_of_contents: true
+---
+
+Wing is a cloud-oriented programming language we’re building that aims to break
+down the barriers that get in the way of building applications using cloud
+services, and bring back the flow for developers.
+
+One of the main features of Wing is that it distinguishes between "preflight
+code" (code that defines your cloud infrastructure, and runs as part of
+compilation) and "inflight code" (code that runs in the cloud, at runtime). We
```

Comment:
Not sure "code" is needed here, since you're also calling it code right after.

```suggestion
One of the main features of Wing is that it distinguishes between "preflight" 
(code that defines your cloud infrastructure, and runs as part of
compilation) and "inflight" (code that runs in the cloud, at runtime). We
```